src/index.js

```diff
@@ -22,6 +74,8 @@ $('#manager-button').click(() => {
 
 $('#login-button').click(() => {
   if ($('#username-input').val().includes('customer') && $('#password-input').val() === 'overlook2019') {
+    guestId = $('#username-input').val().split('r')[1];
```

Comment:
This is a super interesting way of doing this, I was starting to do the same thing. I found out that there's another way to get the numbers out of a string using Regex, it's super bizarre but also works haha 
<img width="404" alt="Screen Shot 2019-11-03 at 6 16 35 PM" src="https://user-images.githubusercontent.com/51575613/68095381-29a04780-fe66-11e9-925f-d46646814db5.png">